### PR TITLE
chore(EMS-662): update README.md

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds more details about EXIP, the code structure/approach and some development principles.

Link to the markdown generation for easier reading: https://github.com/UK-Export-Finance/exip/blob/603970f99be82a24c6dff4fb120fa35dd940ae2e/README.md

### CODE REVIEW

**Review by abhi-markan — APPROVED** [review]
👍🏼 LGTM
Good effort @ttbarnes for an improved documentation, once I have finished all `CI/CD` and `IaC` tickets I will use the similar template and will update accordingly.

Well done.

**Comment by ttbarnes** [comment]
Thanks @abhi-markan most appreciated - especially useful for new starters 😊 Sounds great re CI/CD/IaC.

Found a couple of typos with fresh eyes, will fix asap. Also ideally would like @Zainzzkk and @IgnatG eyes on this before merging 🙏

File: README.md [hunk 1]
```diff
@@ -30,11 +37,22 @@ This repo is based on [template-typescript-package](https://github.com/UK-Export
 1. Execute `docker-compose up` (requires `--build` for first time usage) from the root directory
 2. Visit [http://localhost:5000](http://localhost:5000) in your browser
 
-Alternatively, run via npm in the UI:
+Alternatively, run via npm in each microservice.
+
+In one terminal:
 
 1. `cd src/ui`
 2. `npm start`
```
Comment (Zainzzkk) [inline README.md:45]: we can also do `npm run dev` with hot reloading

File: README.md [hunk 2]
```diff
@@ -30,11 +37,22 @@ This repo is based on [template-typescript-package](https://github.com/UK-Export
...
 
+In another terminal:
+
+1. `cd src/api`
+2. `npm start`
+
+For hot reloading, run execute `npm run dev` instead of `npm start`.
```
Comment (Zainzzkk) [inline README.md:52]: can we add this as point 3 in both, so it is a little clearer

File: README.md [hunk 3]
```diff
@@ -66,25 +84,240 @@ npx cypress run --spec "cypress/e2e/journeys/example.spec.js" --config video=fal
...
+For controllers/pages that are not part of the Quote tool or Insurance, e.g "problem with this service", use [corePageVariables](https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/helpers/page-variables/core/index.ts). Example usage:
+
+```js
+const get = (req: Request, res: Response) =>
+  res.render(
+    TEMPLATES.PROBLEM_WITH_SERVICE,
+    corePageVariables({
+      PAGE_CONTENT_STRINGS: PAGES.EXAMPLE,
+      BACK_LINK: req.headers.referer,
+      PRODUCT: { DESCRIPTION: PRODUCT.DESCRIPTION.GENERIC },
```
Comment (Zainzzkk) [inline README.md:234]: core page variables also has a START_ROUTE.  pedantic but can we add it here

Comment (ttbarnes) [reply README.md:234]: Good spot, added this

**Review by Zainzzkk — APPROVED** [review]
LGTM

**Pull request merged**